Made autotests for the Stellar Burger project

Smuhler opened the pull request.

Smuhler committed:
Made autotests for the Stellar Burger project

AlexKitIt commented on conftest.py:
```diff
@@ -0,0 +1,22 @@
+password = 'vsem privet'
+
+
+@pytest.fixture
+def create_login():  # Генератор логина
+    i = random.randint(1000, 9999)
+    return f'alexeyvolkov9{i}@yandex.ru'
+
+
+@pytest.fixture
+def create_password():  # Генератор пароля
+    return f'{random.randint(100000, 9999999)}'
+
+
+@pytest.fixture
+def incorrect_password():  # Генератор некорректного пароля
+    return f'{random.randint(1, 99999)}'
```
Нужно исправить: фикстуры не занимаются прокидыванием данных в тест, они выполняют сложную логику предусловий\постусловий. Это стоит перенести в helpers

AlexKitIt commented on tests/test_account_link.py:
```diff
@@ -0,0 +1,13 @@
+class TestAccountLink:
+
+    def test_account_link(self):
+        driver = webdriver.Chrome()
```
Нужно исправить:здесь и далее: инстанс драйвера должен подниматься и закрываться фикстуре

AlexKitIt commented on tests/test_register_page.py:
```diff
@@ -0,0 +1,36 @@
+        driver.find_element(By.XPATH, register_password_xpath).send_keys(incorrect_password)
+        driver.find_element(By.XPATH, register_button_xpath).click()
+        WebDriverWait(driver, 3).until(
+            expected_conditions.visibility_of_element_located((By.XPATH, './/p[text()="Некорректный пароль"]')))
+        assert 'input__placeholder-filled' in driver.find_element(By.XPATH, './/label[text()="Пароль"]').get_attribute(
+            'class') and driver.find_element(By.XPATH,
+                                             './/label[text()="Пароль"]/parent::div/parent::div/p').text == 'Некорректный пароль'
+        driver.quit()
```
Нужно исправить: здесь и далее: все локаторы хранятся отдельно от тестов в модуле locators

Smuhler committed:
Many fixes for review

Rename constructor.py to helpers.py
transferred helpers from conftest.py
transferred locators from tests to locators path
added initialize driver in conftest.fixture

Smuhler committed:
Update README.md

Smuhler committed:
Fix locators

Smuhler committed:
added logo link test
